fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -317,6 +323,10 @@
     private static Predicate<Property> hasFedoraNamespace =
         p -> !p.isAnon() && p.getNameSpace().startsWith(REPOSITORY_NAMESPACE);
 
+    // DOAP release decribes fedora build information
```

Comment:
May as well move this property up closer to the `DOAP namespace`... instead of down here at the bottom of the file.